<launch>

  <node ns="camera0" pkg="cv_camera" name="cam0" type="cv_camera_node" output="screen">
    <param name="rate" value="30" />
    <param name="device_id" value="0" />
    <param name="frame_id" value="camera_primer" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <!-- <param name="camera_info" value="???" /> -->
    <!-- <param name="file" value="???" /> -->
  </node>

  <node ns="opencv_tracking" name="cam0_tracking" pkg="camera_tutorials" type="tracking_node.py" />
  <!-- <node ns="test_vision" pkg="camera_tutorials" name="cam0_convert" type="test_vision_node" /> -->
</launch>
